src/core/jay/jay_generated.h

```diff
@@ -23,8 +23,10 @@ struct StatsFloat32;
 struct StatsFloat64;
 
 struct Frame;
+struct FrameBuilder;
```

Comment:
How this `jay_generated.h` file is generated?

Reply:
There is a comment about this in `jay.fbs`: you run `flatc --cpp jay.fbs`, where `flatc` must be installed through either `brew` or `apt-get` or whatever mechanism is available on your system.